test/integration/docker.el

```diff
@@ -0,0 +1,47 @@
+(require 'test/common)
+
+(defvar eldev--docker-emacs-version "27.2")
+
+(ert-deftest eldev-docker-emacs-1 ()
+  (skip-unless (and (eldev-docker-executable nil)
+                    (eldev--docker-on-supported-os)))
+  (let ((eldev--test-eldev-local (expand-file-name default-directory)))
+    (eldev--test-run "trivial-project"
+        ("--quiet"
+         "docker"
+         eldev--docker-emacs-version
+         "emacs"
+         "--batch"
+         "--eval"
+         `(prin1 (+ 1 2)))
+        (should (string-suffix-p "3" stdout))
+        (should (= exit-code 0)))))
+
+(ert-deftest eldev-docker-emacs-2 ()
```

Comment:
I've added a new test against Emacs `24`, I thought it might be useful to test against the oldest supported version.  Let me know if you think this is overkill and I'll remove it.